README.md

```diff
@@ -0,0 +1,27 @@
+# Cookes and Sessions
+
+## Objective
+
+We're going to make a very simple shopping cart, stored entirely in the session.
+
+## Introduction
+
+The Rails `session` method gives us access to the Rails session. The session is a datastore implemented with cookies. You can store simple data structures in the session. ActiveRecord models, no. Arrays of strings or numbers, yes. Basically, if it could be converted into JSON, it can go in the `session`.
```

Comment:
They don't quite know what JSON is. Maybe just say keep it to hashes, arrays, strings and numbers